src/NameFieldSet.tsx

```diff
@@ -25,14 +25,19 @@ export class NameFieldSet extends React.Component<INameFieldSetProps, INameField
   public render() {
     const names = this.state.names.map((value: string, index: number) => {
       let minusBtn = null;
+      let autoFocus = false;
+      const namesLength = this.state.names.length;
+      if ((namesLength === 2 && index === 0) || (namesLength > 2 && index === namesLength - 1)) {
```

Comment:
Replace this by:

const autoFocus = (namesLength === 2 && index === 0) || (namesLength > 2 && index === namesLength - 1);